src/Sylius/Bundle/ApiBundle/ApiPlatform/Bridge/Symfony/Routing/CachedRouteNameResolver.php

```diff
@@ -47,6 +47,7 @@ public function getRouteName(string $resourceClass, $operationType /*, array $co
 
         return $this->getCached($cacheKey, function () use ($resourceClass, $operationType, $context) {
             /** @psalm-suppress TooManyArguments */
+            /** @phpstan-ignore-next-line */
             return $this->decorated->getRouteName($resourceClass, $operationType, $context);
```

Comment:
regarding static checks I think this will fix the issue

```suggestion
            /** @psalm-suppress TooManyArguments */
            return $this->decorated->getRouteName($resourceClass, $operationType, $context); // @phpstan-ignore-line
```